cubids/data/config.yml

```diff
@@ -1,28 +1,38 @@
 # These are non-BIDS fields that can be added by CuBIDS
 derived_params:
```

Comment:
I feel like this config file could be simplified. For one thing, is `suggest_variant_rename` ever `no`?

Reply:
We've set it to no occasionally. It's for when you want to know if a scan differs on some property but don't think it's worth adding it to the file name

Reply:
Ah, that's really good to know. It still seems like the overall file could be shorter, but dropping that field def won't work.